### PR TITLE
test reusable

### PR DESCRIPTION
**Pull request by pshynin**

### CODE REVIEW

File: .github/workflows/build-workflow.yml [hunk 1]
```diff
@@ -42,3 +44,13 @@ jobs:
           TAG_NAME: upstream-3-tag-${{ steps.date.outputs.date }}
           RELEASE_NAME: upstream-3-name-${{ steps.date.outputs.date }}
 
+  upstream-trigger:
+    runs-on: ubuntu-latest
+    needs: [ upstream-3 ]
+    uses: pshynin/gha-test/.github/workflows/test-reusable-workflow.yml@v1
```
Comment (rajington) [inline .github/workflows/build-workflow.yml]:
```suggestion
    uses: pshynin/gha-test/.github/workflows/test-reusable-workflow.yml@test4
```
maybe?

File: .github/workflows/build-workflow.yml [hunk 2]
```diff
@@ -42,3 +44,13 @@ jobs:
           TAG_NAME: upstream-3-tag-${{ steps.date.outputs.date }}
           RELEASE_NAME: upstream-3-name-${{ steps.date.outputs.date }}
 
+  upstream-trigger:
+    runs-on: ubuntu-latest
+    needs: [ upstream-3 ]
```
Comment (rajington) [inline .github/workflows/build-workflow.yml]:
```suggestion
    needs: [upstream-3]
```

File: .github/workflows/build-workflow.yml (hunk 1 above)
Comment (pshynin) [reply .github/workflows/build-workflow.yml]: well.. this should be some static value i guess, otherwise how it will be triggered on a new branch 🙄

Comment (pshynin) [reply .github/workflows/build-workflow.yml]: maybe this workflow has to be on master first.. then it will work 🤔🤔🤔

Comment (rajington) [reply .github/workflows/build-workflow.yml]: yeah that makes sense, just saying that `v1` tag doesn't exist https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_iduses

**Pull request merged**